### ISSUE TITLE
Roundcube user & DB missing in postgresql image

### ISSUE THREAD

**Issue by muhlemmer** [Mailu/Mailu#1471]
#1268 added PostgreSQL support to roundcube. I generate a new config using Setup Utility and select the Mailu provided PostgreSQL image. This sets `DB_FLAVOR=postgresql`. In turn, Roundcube also goes in PostgreSQl mode and tries to connect to the database service with defaults:

https://github.com/Mailu/Mailu/blob/5798880f08168507652af7ab9e9afb4e6ecdcc0a/webmails/roundcube/start.py#L25-L28

However, the PostgreSQL startup script does not create this user and DB:

https://github.com/Mailu/Mailu/blob/5798880f08168507652af7ab9e9afb4e6ecdcc0a/optional/postgresql/start.py#L14-L26

A choice has to be made:

1. Extend the script (and `pg_hba.conf`) to add this user and DB. Slight increase in complexity, while we are looking to retire the image.
2. Change the default roundcube user / DB to mailu & mailu to use current settings. Does Roundcube use prefixed tables?
3. Start shipping stock PostgreSQL image asap and cleanup the mess. This might take a bit of time and meanwhile this deployment will remain broken. #1216

Opinions welcome. I'll implement, no worries.

PS: If we ship a stock image for PostgreSQL, by default it only creates 1 user and 1 database which names can be set by environment variables. If we need to use more users, some scripts would still be needed. https://github.com/docker-library/docs/blob/master/postgres/README.md#initialization-scripts

cc: @micw @kaiyou

**Comment by dhoppe** [comment]
I just upgraded from 1.7 to Master because I wanted to test Roundcube including CalDAV and Enigmail. 

I would be happy about a solution to this problem and I think Roundcube should use a dedicated database.

**Comment by dhoppe** [comment]
I have solved the problem temporarily by adding the following line to the ```mailu.env``` file:
```
ROUNDCUBE_DB_FLAVOR=sqlite
```
The last commit was almost a month ago, and for a Docker based project, the last release was also a relatively long time ago. 

I also noticed that all Docker images are based on Apline 3.10, which makes tools like Rspamd very outdated. Version 1.9.4 for example was released in May 2019. 

I would like to support you in the development of Mailu, but I do not understand why even small pull requests like #1532 are not merged.

**Comment by Nebukadneza** [comment]
Hi There,

The `Mailu`-Project is currently in a bit of a bind! We are short on man-power, and we need to judge if it is possible for us to put in some work on this issue.

To help with that, we are currently trying to find out which issues are actively keeping users from using `Mailu`, which issues have someone who want to work on them — and which issues may be less important. These a less important ones could be discarded for the time being, until the project is in a more stable and regular state once again.

In order for us to better assess this, it would be helpful if you could put a **reaction on this post** (use the :smiley: icon to the top-right).
- 👍️ if you **need this** to be able to use Mailu. Ideally, you’d also be able to test this on your installation, and provide feedback …
- 🎉 if you find it a **nice bonus**, but no deal-breaker
- 🚀 if you want to **work on it yourself**!
We want to keep this **voting open for 2 weeks** from now, so please help out!

**Comment by parisni** [comment]
In order to get both roundcube and mailu on postgresql one have to add those configs to the env. This might be added to the documentation because currently roundcube crashes fallowing the postgres migration docs
```
ROUNDCUBE_DB_FLAVOR=postgresql
ROUNDCUBE_DB_USER=roundcube
ROUNDCUBE_DB_PW=password
ROUNDCUBE_DB_HOST=host
ROUNDCUBE_DB_NAME=roundcube
```

**Comment by parisni** [comment]
Fixed in #1831
